playbooks/adhoc/atomic_openshift_tutorial_reset.yml

```diff
@@ -8,59 +11,6 @@
   sudo: yes
 
   tasks:
-    - service: name={{ item }} state=stopped
-      with_items:
-        - openvswitch
-        - origin-master
-        - origin-node
-        - atomic-openshift-master
```

Comment:
We could include `*master-api` and `*master-controllers` in the list of services to stop or add them in the HA work.